src/libraries/System.Console/src/System/Console.cs

```diff
@@ -22,7 +22,7 @@ public static class Console
         // there's little benefit to having a large buffer.  So we use a smaller buffer size to reduce working set.
         private const int WriteBufferSize = 256;
 
-        private static object InternalSyncObject = new object(); // for synchronizing changing of Console's static fields
+        private static readonly object InternalSyncObject = new object();
```

Comment:
Nit: Since this is a field, it should have `s_`` prefix according to the coding conventions.

Reply:
Also start with lower case eg  `s_syncObject`